bcipy/signal_model/offline_analysis_m.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from bcipy.helpers.load import load_experimental_data, read_data_csv, load_json_parameters
+from bcipy.signal_processing.sig_pro import sig_pro
+from bcipy.helpers.triggers import trigger_decoder
+from bcipy.helpers.bci_task_related import trial_reshaper
+from bcipy.signal_model.mach_learning.train_model import train_m_estimator_pipeline
+from bcipy.helpers.data_visualization import generate_offline_analysis_screen
+from bcipy.helpers.acquisition_related import analysis_channels
+from time import time
+import pickle
+
+
+def offline_analysis_m(data_folder=None, parameters={}):
```

Comment:
Instead of making another function, let's make a parameter which the regular offline analysis can use to decide which method to use

Reply:
fixed